frontend/tests/Factories/Problem.php

```diff
@@ -257,6 +265,10 @@ public static function createProblem(
             }
             \OmegaUp\DAO\Problems::update($problem);
         }
+        if ($params->qualitySeal === true) {
```

Comment:
nit:

```suggestion
        if ($params->qualitySeal) {
```

los booleanos no necesitan compararse con `true` o `false`: se pueden usar directamente en la condición.